opensearchapi/api.pointintime.create.go

```diff
@@ -0,0 +1,260 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+// The OpenSearch Contributors require contributions made to
+// this file be licensed under the Apache-2.0 license or a
+// compatible open source license.
+//
+// Modifications Copyright OpenSearch Contributors. See
+// GitHub history for details.
+
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
This is new code? We don't need a ES grant, remove this license part below.

Reply:
Question for me as I am not into such license things.
Can I remove the ES grant if it is new code but follows the same scheme and has similarity to code that has ES grant? As the `Withxyz()` functions are all the same.

Reply:
IANAL, but if you _copied_ any code with such license then we should probably keep it. But if it's _new code_ then we don't need it. Check out https://github.com/opensearch-project/.github/blob/main/CONTRIBUTING.md#license-headers for more info.